src/python/pants/engine/selectors.py

```diff
@@ -97,11 +97,15 @@ class SelectDependencies(datatype('Dependencies',
   `dep_product`.
   """
 
-  def __new__(cls, product, dep_product, field='dependencies', field_types=tuple(), transitive=False):
+  DEFAULT_FIELD = 'dependencies'
```

Comment:
Yes. I wanted to have a variable for it so that both places would reference the same thing, rather than using literals in both places. But I can inline them if you think it's not worth the additional complexity.